### ISSUE TITLE
[Bug] Turrets cease to follow movement 

### ISSUE THREAD

**Issue by SabreMania** [tsunamayo/Skywanderers#1780]
### Short Description of the issue:

I have tested many different angles, but while I was not always able to reproduce some of the issues, there is one that remains consistant. If you depress your guns enough that they hit the lower border of your turret, the turret ceases to respond to mice movement until you input a movement that aims 'upwards'. This might not seem like an issue, but if you are tracking a target that is close to your maximum depression angle, this can lead to your turret just stopping to respond to your commands, rather than simply not depress anymore but keep following the horizontal movement. The same happens if you elevate your guns and hit the upper border of your armor.

This is only valid by manually aiming the turret. I have not tested this with an AI.

...

### Steps to reproduce the issue:
Create a turret that is encased in armor, and aim as high or as low as you can, then try to make the turret traverse.
...

### Environment
- Skywanderers version: 20w16c
- Graphics Card: NVIDIA GTX 970

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\TsunamayoCorp\Skywanderers\`

**Comment by tsunamayo** [comment]
Yes indeed that was by design, but I agree it will feel unresponsive. I will see what I can do.
Cheers
